lib/Dispatcher.js

```diff
@@ -111,14 +125,18 @@ module.exports = function () {
      * @param {String|Function} handler The function or name of the method that handles the action
      * @returns {number}
      */
-    Dispatcher._registerHandler = function registerHandler(action, name, handler) {
-        Dispatcher.handlers[action] = Dispatcher.handlers[action] || [];
```

Comment:
If we use a pseudo classical approach the classes should not be generated dynamically